os/os.proto

```diff
@@ -253,14 +284,24 @@ message InstallError {
     UNEXPECTED_SWITCHOVER = 7;
     // Failed to sync the transferred OS package to the standby Supervisor. The
     // detail value MUST have more information.
+    // On systems which install the package on all Supervisors in one command,
```

Comment:
A dual Supervisor Target which requires installing the entire system with one Install RPC, MUST not return SYNC_FAIL.

Reply:
this is missing

Reply:
done.